modules/node_modules/@frogpond/ccci-stolaf-college/v1/jobs/index.js

```diff
@@ -84,15 +117,58 @@ function fixupEmailFormat(email) {
 	}
 }
 
+function nextPageExists(dom) {
+	if (dom === undefined) {
+		return false
+	}
+
+	const elements = dom.window.document.querySelectorAll('.next.page-numbers')
+	return elements.length > 0
+}
+
+export function findPageUrls(dom) {
+	const linkElements =[
+		...dom.window.document.querySelectorAll('.gv-list-view > .gv-list-view-title > h3 > a')
+	]
+
+	return linkElements.map((url) => url.getAttribute('href'))
+}
+
+/**
+ * The wp-json response now provides a much rougher response that requires a lot
```

Comment:
When making a time-sensitive reference, it's nice to include a date just so we don't have to run blame to figure out what "now" refers to.

Reply:
For sure, I'll add additional context to make it clear when things changed.